docs/src/pages/theming.mdx

```diff
@@ -266,7 +266,7 @@ component match your design in the exact way you want.
     className="bg-slate-800 ut-label:text-lg ut-allowed-content:ut-uploading:text-red-300"
                |            |                └─ applied to allowed content when uploading
                |            └─ applied to label
-               └─ applied to container
+               └─ applied to the container
```

Comment:
either revert, or change the ones above as well.

Reply:
Change like what? Can you make the suggestion so I can commit?

Reply:
I can't easily make the recommendations, you'll need to change it in an editor. 

basically: 
 applied to label --> applied to  the label
 applied to allowed --> applied to the allowed
 
It may require some rearranging of the label lines.